de-DE/docs/find.ndb

```diff
@@ -375,6 +375,8 @@ stfa~Bestimmt eine Kurzzeit-Fourieranalyse des 
 str~Definiert den lokalen Zeichenkettenvariablensatz einer Prozedur~procedure local strings~help str
 STR.at(POS)~Gibt das Zeichen aus der Zeichenkette an der Stelle POS zur�ck~methode strings zeichenketten character at()~.at(
 STR.fnd(STR,POS)~Gibt die Position des ersten Auftretens von STR in der Zeichenkette ab dem Zeichen POS zur�ck~methode strings zeichenketten find fnd()~.fnd(
+STR.first~Gibt das erste Zeichen aus einer Zeichenkette zur�ck first()~.first(
+STR.last~Gibt das letzte Zeichen aus einer Zeichenkette zur�ck last()~.last(

```

Comment:
Lacks the documentation of `firstch()` and `lastch()`. There's also the problem of a missing keywords section. You can install the NBD tools plugin for NumeRe, which will give you a GUI for modifying and creating new entries in this file.